content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -79,7 +79,8 @@ different Kubernetes components.
 | `CSIMigrationRBD` | `false` | Alpha | 1.23 | |
 | `CSIMigrationvSphere` | `false` | Alpha | 1.18 | 1.18 |
 | `CSIMigrationvSphere` | `false` | Beta | 1.19 | 1.24 |
-| `CSIMigrationvSphere` | `true` | Beta | 1.25 | |
+| `CSIMigrationvSphere` | `true` | Beta | 1.25 | 1.25 |
+| `CSIMigrationvSphere` | `true`  | GA | 1.26  | -  |
```

Comment:
These lines are supposed to be migrated to the second table in the page, which is for GA'ed or deprecated features.

Reply:
done.